data-structures-linked-lists/get-tail.js

```diff
@@ -0,0 +1,15 @@
+/* exported getTail */
+
+function getTail(list) {
+  if (!list.next) {
+    return list.data;
+  }
+
+  let listCopy = list;
+
```

Comment:
This is not a copy. Objects are pass-by-reference. We don't really need a copy anyway.